mosu-app/src/features/notice/hooks/useFile.ts

```diff
@@ -0,0 +1,30 @@
+import { useRef, useState } from "react";
+
+const useFile = () => {
+    const [files, setFiles] = useState<File[]>([]);
+    const fileInputRef = useRef<HTMLInputElement>(null);
+
+    const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        if (!e.target.files) return;
+        const selectedFiles = Array.from(e.target.files);
+        setFiles((prev) => [...prev, ...selectedFiles]);
```

Comment:
이러면 같은 파일 이름이 중복되어 올라갈 수 있을거 같아요
```suggestion
        const newSelectedFiles = Array.from(e.target.files);
    setFiles((prev) => {
        const uniqueNewFiles = newSelectedFiles.filter(
            (newFile) => !prev.some((existingFile) => existingFile.name === newFile.name)
        );
        return [...prev, ...uniqueNewFiles];
    });
```
만약 중복을 허용하고 싶지 않다면 이런 식으로 중복 여부를 확인하는건 어떨까요?